packages/lit-dev-tools-cjs/src/api-docs/configs/lit-2.ts

```diff
@@ -0,0 +1,194 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as pathlib from 'path';
+
+import type {ApiDocsConfig} from '../types.js';
+
+const root = pathlib.resolve(__dirname, '..', '..', '..', '..', '..');
+const workDir = pathlib.join(root, 'packages', 'lit-dev-api');
+const gitDir = pathlib.join(workDir, 'lit');
+const litDir = pathlib.join(gitDir, 'packages', 'lit');
+const srcDir = pathlib.join(litDir, 'src');
+const outDir = pathlib.join(workDir, 'api-data');
+
+/**
+ * lit.dev API docs configuration for Lit 2.x
+ */
+export const lit2Config: ApiDocsConfig = {
+  repo: 'https://github.com/lit/lit',
+  sha: 'f8ee010bc515e4bb319e98408d38ef3d971cc08b',
+  gitDir,
+  tsConfigPath: pathlib.join(litDir, 'tsconfig.json'),
+  pagesOutPath: pathlib.resolve(outDir, 'pages.json'),
+  symbolsOutPath: pathlib.resolve(outDir, 'symbols.json'),
+  typedocRoot: pathlib.join(root, 'packages'),
+
+  entrypointModules: [
+    pathlib.join(srcDir, 'async-directive.ts'),
+    pathlib.join(srcDir, 'decorators.ts'),
+    pathlib.join(srcDir, 'directives/'), // Entire directory
+    pathlib.join(srcDir, 'directive.ts'),
+    pathlib.join(srcDir, 'directive-helpers.ts'),
+    // Don't include html.ts because it is already re-exported by index.ts.
+    //   pathlib.join(srcDir, 'html.ts'),
+    // Don't include hydration because it's not ready yet.
+    //   pathlib.join(srcDir, 'hydrate.ts'),
+    //   pathlib.join(srcDir, 'hydrate-support.ts'),
+    pathlib.join(srcDir, 'index.ts'),
+    // Don't include polyfill-support.ts because it doesn't export anything.
+    //   pathlib.join(srcDir, 'polyfill-support.ts'),
+    pathlib.join(srcDir, 'static-html.ts'),
+  ],
+
+  symbolOrder: ['LitElement', 'ReactiveElement'],
+
+  pages: [
+    {
+      slug: 'LitElement',
+      title: 'LitElement',
+    },
+    {
+      slug: 'ReactiveElement',
+      title: 'ReactiveElement',
+    },
+    {
+      slug: 'templates',
+      title: 'Templates',
+    },
+    {
+      slug: 'styles',
+      title: 'Styles',
+    },
+    {
+      slug: 'decorators',
+      title: 'Decorators',
+    },
+    {
+      slug: 'directives',
+      title: 'Directives',
+      anchorFilter: (node) => node.kindString === 'Function',
+    },
+    {
+      slug: 'custom-directives',
+      title: 'Custom directives',
+    },
+    {
+      slug: 'static-html',
+      title: 'Static HTML',
+    },
+    {
+      slug: 'controllers',
+      title: 'Controllers',
+    },
+    {
+      slug: 'misc',
+      title: 'Misc',
+    },
+  ],
+
+  pageForSymbol(node): string {
+    const entrypoint = node.entrypointSources?.[0]?.fileName ?? '';
+    if (entrypoint.includes('/directives/')) {
```

Comment:
comment? Is this to put all directives on one page?

Reply:
Yep that's right -- isn't it kinda self-documenting though? The method is documented in the interface (because note there will be multiple of these configs soon).